packages/react-native-renderer/src/ReactNativeTypes.js

```diff
@@ -100,6 +100,46 @@ type SecretInternalsType = {
   ...
 };
 
+type InspectorDataProps = $ReadOnly<{
+  [propName: string]: string,
+  ...,
+}>;
+
+type InspectorDataSource = $ReadOnly<{|
+  fileName?: string,
+  lineNumber?: number,
+|}>;
+
+type InspectorDataGetter = (
+  (componentOrHandle: any) => ?number,
+) => $ReadOnly<{|
+  measure: Function,
+  props: InspectorDataProps,
+  source: InspectorDataSource,
+|}>;
+
+export type InspectorData = $ReadOnly<{|
+  hierarchy: Array<{|
+    name: ?string,
+    getInspectorData: InspectorDataGetter,
+  |}>,
+  selection: ?number,
```

Comment:
What does this value represent? Just curious.

Reply:
It's `fiberHierarchy.indexOf(instance);`, I believe it's used to show which component is inspected in the hierarchy.

Reply:
Yeah, I saw that's what it was being assigned to, I just ... didn't understand its purpose 😅and the name was kind of confusing.

Reply:
Updated to `selectedIndex` 👍